src/context/cdhashmap.h

```diff
@@ -192,7 +192,7 @@ class CDOhash_map : public ContextObj {
          bool allocatedInCMM = false) :
     ContextObj(allocatedInCMM, context),
     d_key(key),
-    d_data(),
+    d_data(data),
```

Comment:
This makes me nervous. The initialization order in this function has been very picky. I'd like to see larger scale tests on just changing this line. (paranoid.)